### ISSUE TITLE
timestamps are missing `logicalType` metadata when written as `Int96`

### ISSUE THREAD

**Issue by ExpandingMan** [dask/fastparquet#672]
Hi, I'm in the process of writing a Parquet implementation for the Julia language.

I'm trying to support `Int96` timestamps, and was confused because I was loading them as bare `Int96` rather than any time type.

I took a look at the metadata, and it looks like `fastparquet` is defining *neither* of
- `logicalType`
- `converted_type`

in the schema metadata when writing files written with the option `times="int96"`.

At first I was convinced this was a bug, but the current spec says frustratingly little about `Int96` other than that, basically, one should support it.

This is of course pretty annoying, but I did notice this curious statement in the thrift schema:
```
   *   INT96 (only used for legacy timestamps) - undefined
```

So, in short, I have two questions for the maintainers:
- Is this a bug, or is it deliberate?
- If it is not a bug, do you have a reference?  I tried digging around in the parquet spec repo a bit but didn't find anything definitive.
